### DISCUSSION TITLE
debian 6.5

### DISCUSSION THREAD

**Discussion by aaronms1** [NVIDIA/open-gpu-kernel-modules#590] [Q&A]
do the latest modules work with debian kernel 6.5.0-13 -amd64 ?

**Comment by ptr1337** [comment]
> Supported Linux Kernel Versions
> The NVIDIA open kernel modules support the same range of Linux kernel versions that are supported with the proprietary NVIDIA kernel modules. This is currently Linux kernel 4.15 or newer.

Yes, it is supported.
